neonvm/runner/main.go

```diff
@@ -676,11 +676,14 @@ func run(logger *zap.Logger) error {
 	}
 
 	// create iso9660 disk with runtime options (command, args, envs, mounts)
-	sysctl := []string{}
+	sysctl := []string{
+		"kernel.core_pattern=/tmp/core.%e.%p.%h.%t",
```

Comment:
Not sure if storing it in `/tmp` is a good idea... we've previously seen issues with kernel panics from tmpfs usage (see #450). Maybe the coredumps are small enough in practice that it'll be fine? WDYT?

Reply:
Right, I changed it back to `/proc/sys/kernel/core_pattern -> core`. That's actually how it was requested, in an issue, so I hope it will be even easier for people to find coredumps.

Reply:
Fair enough, sounds good